core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -1067,11 +1067,14 @@ private[spark] class TaskSetManager(
       }
     }
     for ((tid, info) <- taskInfos if info.running && info.executorId == execId) {
+      // If the task is launching, this indicates that Driver has sent LaunchTask to Executor,
+      // but Executor has not sent StatusUpdate(TaskState.RUNNING) to Driver. Hence, we assume that
+      // the task is not running, and it is NetworkFailure rather than TaskFailure.
```

Comment:
Move the comment to the `!info.launching` case ?

Reply:
Updated.